src/manifolds/FixedRankMatrices.jl

```diff
@@ -408,3 +421,34 @@ function zero_tangent_vector!(
     X.Vt .= zeros(eltype(X.Vt), k, n)
     return X
 end
+
+@doc raw"""
+    embed(p::SVDMPoint)
+
+Embed the point `p` from its `SVDMPoint` representation.
+"""
+
+function embed!(::FixedRankMatrices, q, p)
+    return q .= p.U * Diagonal(p.S) * p.Vt
+end
+function embed(M::FixedRankMatrices{m,n,k}, p) where {m,n,k}
+    q = zeros(m, n)
+    return embed!(M, q, p)
+end
+
+@doc raw"""
+    embed(::FixedRankMatrices, Y, p, X)
+
+Embed the tangent vector `X` at point `p` in `M` from
+its `UMVTVector` representation.
+"""
```

Comment:
Our convention is to document the non-mutating function (even if not defined).

```suggestion
@doc raw"""
    embed(M::FixedRankMatrices, p, X)

Embed the tangent vector `X` at point `p` in `M` from
its [`UMVTVector`](@ref) representation.
"""
embed(::FixedRankMatrices, p, X)

```